windows-installer/docs/INSTALL.md

```diff
@@ -0,0 +1,85 @@
+
+## Configuration
+
+Certbot works essentially with a configuration folder, where all the configuration files and generated certificates are available.
+
+On Windows, this folder is `C:\Certbot`. This can be changed using the `--config-dir` flag on the Certbot CLI.
+
+Within this folder you will find the following folders:
+* `C:\Certbot\live`: contains current versions of issued certificates and keys
+* `C:\Certbot\archive`: contains all current and previous versions of issued certificates and keys
+* `C:\Certificate\logs`: contains Certbot activity logs
+
+Doing backups of `C:\Certbot` folder is everything required to save your Certbot installation.
+
+## Certificates renewals
+
+Renewals are supported by Certbot for Windows and are even already configured. Indeed the Certbot installer created a scheduled task during the installation to execute `certbot renew` twice a day.
+
+All certificates created with Certbot for Windows are then automatically renewed out-of-the-box, providing that required environment to renew is available (eg. credentials, running web server ...).
+
+## A note on manual hooks
+
+As explained by the [manual plugin documentation](https://certbot.eff.org/docs/using.html#manual), you may need to setup authentication and cleanup hooks to make the `manual` plugin work as expected, in particular with `DNS-01` challenges.
+
+Similarly to Linux, Certbot for Windows supports any file executable for this platform, if it is available in the `PATH`. It can be `.bat`, `.ps` or even executable binaries like `.exe`.
+
+## A note on files security
+
+Assets generated by Certbot, in particular keys, are critical in term of security and should never been exposed to unauthorized accesses. Certbot for Windows leverages DACLs implemented by the NTFS filesystem to enforce this control: keys and sensible materials are readable by default only by the account that runs Certbot, the `Administrators` group and `SYSTEM`.
+
+## Updating Certbot
+
+When a new version of Certbot release is available, please download the new installer for this new version as described in the _Installation_ section, and run it. The installer will update you existing installation of Certbot.
+
+## Getting help, report issues, ask for new features
+
+If you encounter some difficulties to install or use Certbot for Windows, there is a dedicated area in LetsEncrypt support forum that can be reached at _[URL to dedicated category on Certbot (beta) for Windows in the LetsEncrypt Support Forum to be defined]_.
+
+If you find an issue, or want to ask a new feature on Certbot for Windows, do not hesitate to create a new issue in the [Certbot GitHub repository](https://github.com/certbot/certbot/issues). Please do not forget that some existing features for Linux systems are not yet available on Windows, but are planned to be added (see the section _Supported features_).
```

Comment:
to ask for a new feature

Reply:
Should users add a `windows` tag to their issues for Windows-specific or Windows-related questions?

Reply:
I never saw them adding tags to new issues. Is it possible without a RW access on the Git repository?

Reply:
Added the reference to `area: windows`.